src/utility/SElementBase64.h

```diff
@@ -0,0 +1,20 @@
+/*
+    This file is part of the Arduino_SecureElement library.
+
+    Copyright (c) 2024 Arduino SA
+
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+#pragma once
+
+#include <Arduino.h>
+
+namespace arduino { namespace b64 {
+
+    String urlEncode(const byte in[], unsigned int length);
+    String encode(const byte in[], unsigned int length, const char* prefix, const char* suffix);
```

Comment:
why don't we define only encode this way? It seems a bit redundant the definition 
```suggestion
    String encode(const byte in[], unsigned int length, const char* prefix=nullptr, const char* suffix=nullptr);
```

Reply:
this two functions are also using a different alphabet https://datatracker.ietf.org/doc/html/rfc4648#section-5

Reply:
encode is also adding \n so i've renamed it to pemEncode